packages/remix-dev/vite/plugin.ts

```diff
@@ -255,12 +255,20 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
 
   let resolvePluginConfig =
     async (): Promise<ResolvedRemixVitePluginConfig> => {
+      let defaults: Partial<RemixVitePluginOptions> = {
+        serverBuildPath: "build/server/index.js",
+        assetsBuildDirectory: "build/client",
+        publicPath: "/",
```

Comment:
Nice change!
With this change of default `publicPath` from `/build` to `/`, I feel it became closer to what `base` vite config does (`base` default value is also `/` https://vitejs.dev/config/shared-options.html#base).

I think remix still doesn't support custom basename in general, but I was wondering if it makes sense to deprecate `publicPath` remix config and favor `base` option from user vite config at some point.

For the reference:
- https://github.com/remix-run/remix/issues/8052
- https://github.com/remix-run/remix/discussions/2891

Reply:
That's a really good point! I'll keep this in mind when looking at custom `base` support.